### ISSUE TITLE
KITTI 2015 val set

### ISSUE THREAD

**Issue by udaykusupati** [feihuzhang/GANet#64]
I see you talk about a KITTI 2015 validation set in the ablation study section of the paper. Can you provide the split you use please?
Thanks

**Comment by jiady1990** [comment]
> I see you talk about a KITTI 2015 validation set in the ablation study section of the paper. Can you provide the split you use please?
> Thanks

I have the same question. I've read the code of PSMNet. The training set is the first 160 and the validation set is the rest. I want to know if it is the same setting in GANet.

**Comment by feihuzhang** [comment]
It's a random selection for ablation study. You can select by yourself. Nothing special.
